connexion/operations/swagger2.py

```diff
@@ -51,10 +50,6 @@ def __init__(self, api, method, path, operation, resolver, app_produces, app_con
         :param definitions: `Definitions Object
             <https://github.com/swagger-api/swagger-spec/blob/master/versions/2.0.md#definitionsObject>`_
         :type definitions: dict
-        :param parameter_definitions: Global parameter definitions
```

Comment:
There is indeed some discrepancy, but it's a discrepancy in implemented functionality. The `parameter_definitions` here are not actually used. It might be something that we still want to implement, but for now I find it more clear to remove the unused code.

Reply:
Ah yes, didn't realize it was due to difference in functionality.